crates/iota/src/client_commands.rs

```diff
@@ -3047,3 +3051,33 @@ fn parse_emit_option(s: &str) -> Result<HashSet<EmitOption>, String> {
 
     Ok(options)
 }
+
+/// Warn the user if the CLI falls behind the current on-chain protocol
+/// version.
```

Comment:
rustfmt would probably complain, but it should mention that the version just needs to be different, as this is what it does
```suggestion
/// Warns the user if the CLI does not match the version of the current on-chain protocol.
```

Reply:
you're right ... fixed with https://github.com/iotaledger/iota/pull/5042/commits/f5a8675410bf2dc474d34bbd4d52c4a25231c27d